src/main/java/org/sagebionetworks/web/client/jsinterop/CardConfiguration.java

```diff
@@ -0,0 +1,30 @@
+package org.sagebionetworks.web.client.jsinterop;
+
+import java.util.Map;
+import jsinterop.annotations.JsFunction;
+import jsinterop.annotations.JsOverlay;
+import jsinterop.annotations.JsPackage;
+import jsinterop.annotations.JsType;
+import jsinterop.base.Js;
+import jsinterop.base.JsPropertyMap;
+
+@JsType(isNative = true, namespace = JsPackage.GLOBAL, name = "Object")
+public class CardConfiguration {
+
+  String type;
+  double secondaryLabelLimit;
```

Comment:
```suggestion
  long secondaryLabelLimit;
```

Reply:
Did something change, I thought long values were disallowed?  Or perhaps I'm misunderstanding.
From https://www.gwtproject.org/doc/latest/DevGuideCodingBasicsJSNI.html

> The Java long type cannot be represented in JavaScript as a numeric type, so GWT emulates it using an opaque data structure. This means that JSNI methods cannot process a long as a numeric type. The compiler therefore disallows, by default, directly accessing a long from JSNI: JSNI methods cannot have long as a parameter type or a return type, and they cannot access a long using a [JSNI reference](https://www.gwtproject.org/doc/latest/DevGuideCodingBasics.html#DevGuideJavaScriptNativeInterface).

Reply:
Right!  I'm still thinking in JSNI.  I guess I'll leave it, for the small potential performance gain